src/context/AuthContext.js

```diff
@@ -0,0 +1,35 @@
+import React, {createContext, useState} from 'react';
+import {useHistory} from "react-router-dom";
+
+export const AuthContext = createContext({})
+
+function AuthContextProvider ({ children }) {
+    const [isAuth, toggleIsAuth] = useState(false);
+    const history = useHistory();
+
+    function inloggen() {
+        toggleIsAuth(true)
+        console.log("Gebruiker is ingelogd!")
+        history.push('/profile')
+    }
+
+    function uitloggen() {
+        toggleIsAuth(false)
+        console.log("Gebruiker is uitgelogd!")
+        history.push('/')
+    }
+
+    const data = {
+        ingelogd: isAuth,
+        inlogFunction: inloggen,
```

Comment:
Ik krijg weleens terug dat je beter alles in het nederlands kan schrijven. (heb ik zelf nu ook niet gedaan, maar is vaak feedback die ik terug krijg 🙃)